save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/topbar/TopBarLinks.kt

```diff
@@ -79,19 +108,26 @@ private fun topBarLinks() = FC<TopBarLinksProps> { props ->
             div {
                 className = ClassName("dropdown-menu dropdown-menu-right shadow animated--grow-in${if (isDemoDropdownActive) " show" else "" }")
                 ariaLabelledBy = "demoDropdown"
-                val diktatDemoHref = "/${FrontendRoutes.DEMO.path}/diktat"
-                dropdownEntry(null, "Diktat", window.location.href.contains(diktatDemoHref)) { attrs ->
-                    attrs.onClick = {
-                        isDemoDropdownActive = false
-                        navigate(to = diktatDemoHref)
-                    }
-                }
-                val cpgDemoHref = "/${FrontendRoutes.DEMO.path}/cpg"
-                dropdownEntry(null, "CPG", window.location.href.contains(cpgDemoHref)) { attrs ->
-                    attrs.onClick = {
-                        isDemoDropdownActive = false
-                        navigate(to = cpgDemoHref)
-                    }
+
+                demoDropdownEntry(
+                    "Diktat",
+                    "/${FrontendRoutes.DEMO.path}/diktat",
+                    navigate,
+                    deactivateDropdown,
+                )
+                demoDropdownEntry(
+                    "CPG",
+                    "/${FrontendRoutes.DEMO.path}/cpg",
+                    navigate,
+                    deactivateDropdown,
+                )
+                demos.forEach { demo ->
```

Comment:
```suggestion
                demos.take(3).forEach { demo ->
```
Just to look nice for now

Reply:
are you sure, how we will look all of them? I guess it should be done in another way.
will add it for now with a `TODO` comment

Reply:
Yep, that's what I meant

Reply:
added on request part